mirascope/core/base/stream.py

```diff
@@ -374,7 +379,9 @@ async def generator() -> (
                     ]
                 ):
                     async for chunk, tool in handle_stream_async(
-                        await create(stream=True, **call_kwargs), tool_types
+                        await create(stream=True, **call_kwargs),
+                        tool_types,
+                        allow_partial_tool=partial_tools,
```

Comment:
nit: i don't like that the stream handlers call this `allow_partial_tool` and then the argument here is `partial_tools` we should make this consistent

let's just make them all `partial_tools`? or make them all `allow_partial_tools` up to you